packages/pdf-viewer/pdfSource.test.ts

```diff
@@ -0,0 +1,41 @@
+import { PdfData } from './pdfSource';
+import * as pdfjsLib from 'pdfjs-dist';
+
+const path = require('path');
+const fs = require('fs');
+
+pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/legacy/build/pdf.worker.entry');
+
+const pdfFilePath = path.resolve('config/welcome.pdf');
+
+describe('pdfData', () => {
+	const file = new Uint8Array(fs.readFileSync(pdfFilePath));
+	test('Should have correct page count', async () => {
+		const pdf = new PdfData();
+		await pdf.loadDoc(file);
+		expect(pdf.pageCount).toBe(1);
+	});
+
+	test('Should throw error on invalid file', async () => {
+		const pdf = new PdfData();
+		await expect(async () => {
+			await pdf.loadDoc('');
+		}).rejects.toThrowError();
```

Comment:
Does that actually work? I didn't know it was possible to await an expect call?

Reply:
I did find a few examples like this in the documentation if I'm looking at the right place, https://jestjs.io/docs/asynchronous#asyncawait